### PR TITLE
Swagger fix (no JIRA)

### PR DESCRIPTION
**Pull request by aednichols**
I updated `MethodConfiguration` to match Rawls, but Orch has an additional `NewMethodConfigIngest` that also needs updating.

---

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
👍 Matches rawls.

**Comment by aednichols** [comment]
jenkins retest

**Review by kyuksel — APPROVED** [review]
Swagger works 👍

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -5108,6 +5108,13 @@ definitions:
       methodRepoMethod:
         type: object
         description: Method ID - Map
+      methodConfigVersion:
+        type: integer
+        description: 'Version number, incremented on edit'
+      deleted:
+        type: boolean
+        default: false
+        description: 'Is this config marked as deleted?'
```
Comment (davidangb) [inline src/main/resources/swagger/api-docs.yaml:5117]: add `required: true` for these.

**Review by davidangb — APPROVED** [review]
:+1:

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (aednichols) [reply src/main/resources/swagger/api-docs.yaml:5117]: 👍 fixed!

**Pull request merged**